storage/src/metadata/storage.rs

```diff
@@ -137,8 +139,8 @@ impl<B: Blob, E: Clock + Storage<B> + Metrics> Metadata<B, E> {
         let mut cursor = 16;
```

Comment:
Can you change this `16` to use `u128::SERIALIZED_LEN`?

<img width="725" alt="image" src="https://github.com/user-attachments/assets/ce1a9ca6-5d9b-480a-a4e6-a31d95cf3753" />